catalog/spatial/geocoding/spatial-geocoding-api/src/main/java/org/codice/ddf/spatial/geocoding/Suggestion.java

```diff
@@ -11,19 +11,26 @@
  * License is distributed along with this program and can be found at
  * <http://www.gnu.org/licenses/lgpl.html>.
  */
-package org.codice.ddf.spatial.geocoding.query;
```

Comment:
This is a confusing "change".... it is claiming the old test file now became the new Suggestion class?

Reply:
I agree it's confusing. Even though I've split it into two commits - git still thinks it is a rename